pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -151,6 +153,8 @@ func (config *V1RegistriesConf) Nonempty() bool {
 
 // V2RegistriesConf is the sysregistries v2 configuration format.
 type V2RegistriesConf struct {
+	// The credential store for registries
+	CredsStore []string   `toml:"creds_store"`
```

Comment:
Why is this a list? How are the users expected to use that, what happens on errors?

Reply:
The design mentioned `failover list that would attempt to use each credential helper until one works. https://docs.google.com/document/d/1mTPI9GCvy3C_c11Z1MNQzOR6N_lMBVLHW3gpOllC6rw/edit
Changed to a string type. Maybe the error can be silently ignored this way, the usage seems underministic. I am ok to not use the list, like the credsStore in config.json

Reply:
Thanks, I’ll return to that document.